curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bcc8ccc976fd791610f43.md

```diff
@@ -10,6 +10,38 @@ dashedName: what-is-the-css-transform-property
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What is the CSS transform property, and how does it work?
+
+The CSS transform property is a powerful tool that allows you to modify the visual presentation of elements on  your webpage without affecting the layout of other elements. It enables you to apply various transformations to elements such as rotating, scaling, skewing or translating, moving them in 2D or 3D space.
+
+The transform property works by applying a mathematical transformation to an elements coordinate system. This means you can manipulate an element's shape and position while keeping it's original place and document flow intact.
+
+Let's explore some common transform functions. Here's an example of a box element, we have set the body to have a solid black border so that you can see the box element nested inside the body element. 
+
+The translate function moves an element from its current position. Here's an updated example using the translate function. This CSS rule will move the element with the class box 50 pixels to the right and 100 pixels down from its original position. Here's what the modified result looks like on the screen.
+
+The rotate function rotates an element around a fixed point and this is an example of using the rotate function for the box element from earlier. This will rotate the element forty five degrees clockwise and here's what it looks like.
```

Comment:
```suggestion
The `rotate` function rotates an element around a fixed point and this is an example of using the rotate function for the box element from earlier. This will rotate the element forty five degrees clockwise and here's what it looks like.
```

Reply:
same issue here. Not sure what to do with that `"here's what it looks like"` since there is nothing to look at in this script